### ISSUE TITLE
Freezes after long runtime

### ISSUE THREAD

**Issue by CreeperChief** [mccxiv/tc#493]
The program froze after I left it open in the background for 1-2 hours. I managed to close it after I quit it by pressing Quit Tc in the icon section near the clock but it took 3 minutes to quit the program.

**Comment by Bauke** [comment]
I'm not sure how actionable this issue is, my guess would be that there's some kind of memory leak somewhere and it ended up freezing Tc. I've had Tc running myself for days at a time without issue and never had it freeze up myself.

**Comment by mccxiv** [comment]
Yeah I tried so hard to look for it, and caught a few, but AngularJS has many footguns when destroying and creating views. 
Maybe these days I'd be better at debugging it 🤔
